boards/hail/src/main.rs

```diff
@@ -156,8 +162,12 @@ unsafe fn set_pin_primary_functions() {
 
     PB[00].configure(Some(A)); // SENSORS_SDA - TWIMS1 SDA
     PB[01].configure(Some(A)); // SENSORS_SCL - TWIMS1 SCL
-    PB[02].configure(Some(A)); // A2 - ADC3
-    PB[03].configure(Some(A)); // A3 - ADC4
+                               // Analog Comparator Mode
+    PB[02].configure(Some(E)); // A2 - ACIFC ACBN0
+    PB[03].configure(Some(E)); // A3 - ACIFC ACBP0
+                               // // ADC Mode
+                               // PB[02].configure(Some(A)); // A2 - ADC3
+                               // PB[03].configure(Some(A)); // A3 - ADC4
```

Comment:
Please revert the changes to Hail so we can leave the default operation of those pins as ADC pins (which is what they are labeled as on the silkscreen).